api/data_explorer/controllers/export_url_controller.py

```diff
@@ -1,10 +1,24 @@
 import connexion
 import six
+import json
 
 from flask import current_app
 
 from data_explorer.models.export_url_response import ExportUrlResponse  # noqa: E501
 
 
 def export_url_post():  # noqa: E501
-    return ExportUrlResponse(url='EXPORT_URL')
+    requests = []
+    for table_name in current_app.config['TABLE_NAMES']:
+        # Treat table names as BigQuery tables if they conform to project_id.dataset.table_id
+        if len(table_name.split(".")) == 3:
+            requests.append(
+                json.dumps({
+                    "name": table_name.split(".")[2],
+                    "entityType": "BigQueryTables",
+                    "attributes": {
+                        "TableID": table_name
```

Comment:
As I said earlier, please make this:
```
"table_name": table_name
```
In BigQuery terminology, it's "table name", not "table ID".

Reply:
Done